config.sample.yaml

```
@@ -53,6 +54,8 @@ timezone: ""
 nodes:
   # (Required) CIDR your nodes are on (e.g. 192.168.1.0/24)
   host_network: ""
+  # (Optional) Talos only: If your gateway is not on the .1 address of above CIDR you can set it here (e.g. 192.168.1.254)
```

```suggestion
  # (Optional) If your gateway is not on the .1 address of above CIDR you can set it here (e.g. 192.168.1.254)
```

This actually affects k3s as well